packages/common/src/dto/application-info.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export interface ApplicationInfo {
+  url: string;
+  title: string;
+  icon: string;
+  group?: string;
```

Comment:
what is a group?

Reply:
I see. Makes sense.
It seems to be consistent with what OpenShift Console does with ConsoleLink
![Screenshot_20211019_172932](https://user-images.githubusercontent.com/5887312/137931341-955cab56-28bc-4931-bdf4-a907a12e0571.png)